dataplane/linux/xdp_state.go

```diff
@@ -2155,8 +2155,8 @@ type xdpMemberCache struct {
 
 func newXDPMemberCache(family bpf.IPFamily, bpfLib bpf.BPFDataplane) *xdpMemberCache {
 	return &xdpMemberCache{
-		family:                 family,
-		cache:                  make(map[string]map[bpf.CIDRMapKey]uint32),
+		family: family,
```

Comment:
I don't know why my go fmt kept doing this.

Reply:
Weird. What version of go do you have?

Reply:
1.11.2, I should update.